scarb/src/core/registry/client/cache.rs

```diff
@@ -154,6 +175,28 @@ impl<'c> RegistryClientCache<'c> {
             })
             .await
     }
+
+    /// Check whether the tarball has already been downloaded in the _dl_ directory.
+    /// If user modifies downloaded archive, it's their problem.
```

Comment:
> If user modifies downloaded archive

Wouldn't checksum verification fail in this instance?

Reply:
Exactly

Reply:
#906 changes the code, so that checksums are not verified for already downloaded packages, so this comment now will hold exactly as it says